### PR TITLE
Add new link to redirect users to user guide

### PR DESCRIPTION
**Pull request by seantehds**
The link that is shown to users when they input either the help command or access the help tab has been updated to show our current user guide.

### CODE REVIEW

**Review by JCSnap — APPROVED** [review]
Looks good!

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -46,21 +46,21 @@ public HelpWindow() {
...
+     *                               <li>
+     *                               if this method is called during animation or layout processing.
+     *                               </li>
+     *                               <li>
+     *                               if this method is called on the primary stage.
+     *                               </li>
+     *                               <li>
+     *                               if {@code dialogStage} is already showing.
+     *                               </li>
+     *                               </ul>
```
Comment (asdfghjkxd) [inline src/main/java/seedu/address/ui/HelpWindow.java:63]: Looks good! But should the HTML tags be formatted like this?

**Review by asdfghjkxd — APPROVED** [review]
Looks good overall! There are some concerns regarding the alignment and format of the HTML tags, but otherwise, the code looks fine to me!

**Review by maypfv — COMMENTED** [review]
Looks good to me!

**Pull request merged**